### PR TITLE
Improve routing names

### PR DESCRIPTION
**Pull request by jotjern**
I've improved the routing names by translation groups to Norwegian and using the group names instead of IDs in the URL.

### CODE REVIEW

**Review by Terbau — CHANGES_REQUESTED** [review]
What happens if two groups share the same short name? A fix for this problem would be to set name_short field in the database to be unique.

File: frontend/src/views/groups/GroupsView.tsx [hunk 1]
```diff
@@ -12,27 +12,40 @@ export const GroupsView = () => {
...
 
   const { data: user } = useQuery({
     queryKey: ["groupsData"],
     queryFn: () =>
       axios.get(ME_URL).then((res: AxiosResponse<User>) => {
-        setUser({ user_id: res.data.user_id });
-        return res.data;
+        const user = res.data;
+        user.groups.sort((a, b) => a.name_short.localeCompare(b.name_short));
+        const duplicateNames = new Map<string, number>();
```
Comment (Terbau) [inline frontend/src/views/groups/GroupsView.tsx:24]: I don't really see a reason why to include this part of the code anymore. There shouldn't ever be a duplicate `name_short` coming from the backend after setting the table field to unique.

**Review by Terbau — CHANGES_REQUESTED** [review]

File: frontend/src/views/groups/GroupsView.tsx (hunk 1 above)
Comment (jotjern) [reply frontend/src/views/groups/GroupsView.tsx:24]: Thought i removed it. will fix

File: backend/app/migrations/0001_initial_schema.sql [hunk 2]
```diff
@@ -10,7 +10,7 @@ CREATE TABLE IF NOT EXISTS groups (
 	group_id SERIAL PRIMARY KEY,
 	ow_group_id INTEGER UNIQUE,
 	name TEXT NOT NULL UNIQUE,
-	name_short TEXT,
+	name_short TEXT NOT NULL UNIQUE,
```
Comment (Terbau) [inline backend/app/migrations/0001_initial_schema.sql:13]: Did you forget to add case insensitive unique in the schema?

Comment (Terbau) [inline backend/app/migrations/0001_initial_schema.sql:13]: Did you forget to add case insensitive unique?

**Review by Terbau — CHANGES_REQUESTED** [review]

**Review by Terbau — APPROVED** [review]
lgtm

**Pull request merged**